vision/module_in_frame/ModuleInFrame.py

```diff
@@ -0,0 +1,68 @@
+"""
+This file contains the ModuleInFrame function to detect if the module is in an image
+"""
+
+import cv2
+import numpy as np
+
+def ModuleInFrame(imgPath):
```

Comment:
Refactor so that the image comes in as a numpy array, in production we will not be saving images.